src/System.Net.Security/src/System/Net/SecureProtocols/_SecuritySafeHandles.Windows.cs

```diff
@@ -218,117 +218,6 @@ public new IntPtr DangerousGetHandle()
             throw NotImplemented.ByDesign;
         }
 #endif
-
-        #if false //TODO if not used in Nego Stream as well, please remove it.
```

Comment:
No, of course not. @rajansingh10 @vijaykota please _do not_ just delete or remove code from the Windows implementation. Instead move it to another place or leave it within `#if` pragmas.
Please read my comments in the original review.

Note that the current design may not work for NetworkInformation. We'll most likely need to keep the Windows code as-is (in the original implementation) and write a completely _separate_ PAL on top.